tests/integ/scala/test_dataframe_writer_suite.py

```diff
@@ -209,24 +209,45 @@ def test_write_with_target_column_name_order_all_kinds_of_dataframes(
 
 
 def test_write_table_names(session, db_parameters):
-    schema = quote_name(db_parameters["schema"])
-
-    def create_and_append_check_answer(table_name, full_table_name=None):
+    schema = session.get_current_schema().replace('"', "")
+    database = session.get_current_database().replace('"', "")
```

Comment:
you might want to create a temp schema here. tests are failing because `Schema 'TESTDB_SNOWPARK_PYTHON.PUBLIC' does not exist or not authorized.`

Reply:
though the test failures are not related to temp schema but a privilege issue (it is fixed), but I updated the tests to create and drop new schemas